readme/apps/publish_note.md

```diff
@@ -25,3 +25,9 @@ From there, clicking on the "Share" icon will unpublish the note.
 Using Joplin Cloud Pro or Teams, you have the ability to customise the banner that is displayed at the top of the notes. That includes changing the logo, title or colour of the banner. To do so, open your [Profile page](https://joplincloud.com/users/me) and go to the "Published note banner" section:
 
 <img src="https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/publish_note/Banner.png" width="75%"/>
+
+### Video Tutorial
+
+You can watch the full video tutorial here:
```

Comment:
Add a more descritive text please, for example "For a video tutorial on how to publish a note and customise the banner please follow this link:"